docs/cluster-management/running-locally.md

```diff
@@ -20,3 +21,98 @@ You will be prompted to enter your Client ID and Client Secret. Afterwards, type
 
 [docker]: https://www.docker.com/products/docker
 [docker-compose]: https://www.docker.com/products/docker-compose
+
+## Configuring SD-in-a-Box
+
+SD-in-a-box was intended to be an easy way to run a Screwdriver cluster locally on your development machine so you can demo its features first-hand.
+
+### Custom Docker Images
+
+Since it's powered by Docker, you can determine which images to use for it
```

Comment:
Good idea.

We can add a list of the specific docker images with some short explanation. Like

* screwdrivercd/screwdriver - Screwdriver API. The main engine of the CI/CD cluster.
* screwdrivercd/ui - Screwdriver UI. To pleasantly interact with Screwdriver
* screwdrivercd/store - Screwdriver artifact repository. Responsible for artifacts like logs & templates